jobs/gorouter/spec

```diff
@@ -313,6 +313,9 @@ properties:
   router.logging.syslog_tag:
     description: "Tag to use when writing syslog messages"
     default: "vcap.gorouter"
+  router.logging.syslog_message_limit:
```

Comment:
Can you add more info here about what "0" means and the units that this limit is in?